Description: Fix Moscow tz check
 It seems that the test fails because Moscow tz changed. This patch removes
 the UTC +04 check.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: http://bugs.debian.org/775636
Forwarded: no
Last-Update: 2014-01-21

--- horizon-2015.1~b1.orig/openstack_dashboard/dashboards/settings/user/tests.py
+++ horizon-2015.1~b1/openstack_dashboard/dashboards/settings/user/tests.py
@@ -27,7 +27,7 @@ class UserSettingsTest(test.TestCase):
         res = self.client.get(INDEX_URL)
 
         self.assertContains(res, "UTC +11:00: Australia (Melbourne) Time")
-        self.assertContains(res, "UTC +04:00: Russia (Moscow) Time")
+        self.assertContains(res, "Russia (Moscow) Time")
         self.assertContains(res, "UTC -03:00: Falkland Islands Time")
         self.assertContains(res, "UTC -10:00: United States (Honolulu) Time")
 
